content/en-us/art/characters/facial-animation/moods.md

```diff
@@ -89,7 +89,7 @@ local Players = game:GetService("Players")
 local function onCharacterAdded(character)
 	local humanoid = character:WaitForChild("Humanoid")
 	local animateScript = character:WaitForChild("Animate")
-	animateScript.mood.Animation1.AnimationId = "rbxassetid://7715145252"    -- Mood
+	animateScript.mood.Animation1.AnimationId = "rbxassetid://7715145252" -- Mood
```

Comment:
```suggestion
	animateScript.mood.Animation1.AnimationId = "rbxassetid://7715145252"  -- Mood
```

Reply:
Generally, I prefer two spaces for an inline comment, although there likely isn't a bunch of consistency on this throughout the docs and from different authors.